tests/runner.py

```diff
@@ -1194,6 +1194,13 @@ def show():
            % (expected))
   atexit.register(show)
 
+def suite_for_module(module):
+  import parallel_runner
```

Comment:
It occurred to me that there exists this great tool where something something organize information something useful.

Looks like your general opinion is the recommended one: https://www.python.org/dev/peps/pep-0008/#imports
And sharing an interesting discussion around imports, how they work, etc: http://stackoverflow.com/questions/1699108/local-import-statements-in-python . TIL there's a python disassembler module, imports broadly work the way I think they do, and import statements have a small performance hit due to the lookup.